### PR TITLE
Remove patches to discovery v5

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

**Comment by dshulyak** [comment]
images:

- statusteam/bootnode:original-discovery-v5
- statusteam/status-go:original-discovery-v5

File: vendor/github.com/ethereum/go-ethereum/p2p/discv5/net.go [hunk 1]
```diff
@@ -1066,11 +1066,7 @@ func (net *Network) handle(n *Node, ev nodeEvent, pkt *ingressPacket) error {
 func (net *Network) checkPacket(n *Node, ev nodeEvent, pkt *ingressPacket) error {
 	// Replay prevention checks.
 	switch ev {
-	case pingPacket:
-		if pkt.data.(*ping).Version != Version {
-			return fmt.Errorf("version mismatch")
-		}
-	case findnodeHashPacket, neighborsPacket:
+	case pingPacket, findnodeHashPacket, neighborsPacket:
```
Comment (adambabik) [inline vendor/github.com/ethereum/go-ethereum/p2p/discv5/net.go:1058]: I think only this one and...

File: vendor/github.com/ethereum/go-ethereum/p2p/discv5/udp.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
-const Version = 55
+const Version = 4
```
Comment (adambabik) [inline vendor/github.com/ethereum/go-ethereum/p2p/discv5/udp.go:35]: ...this one are important. If other changes improve some service properties and do not break compatibility with the original version, I think they can stay.

Comment (dshulyak) [reply vendor/github.com/ethereum/go-ethereum/p2p/discv5/udp.go:35]: you are right that other changes don't break compatibility, but i think they are only marginally improving discovery reliability/speed which is not that important for nodes running on PC or VPS.

i will try to get some meaningful metrics from running proxy with and without those changes.

Comment (dshulyak) [reply vendor/github.com/ethereum/go-ethereum/p2p/discv5/udp.go:35]: i think we should remove those changes, discovery v5 will now depend on much bigger network. and having faster invalidation doesn't help much in this case

Comment (adambabik) [reply vendor/github.com/ethereum/go-ethereum/p2p/discv5/udp.go:35]: Ok, let's do that! These adjustments were only for mobile anyway.

**Review by adambabik — APPROVED** [review]

**Pull request merged**